src/pudl/metadata/resources/eiaaeo.py

```diff
@@ -0,0 +1,514 @@
+"""Table definitions for EIA AEO resources."""
+
+from typing import Any
+
+# 2024-04-24: workaround so we can *define* a bunch of schemas without
+# documenting them or expecting assets to exist
+_STAGING_RESOURCE_METADATA: dict[str, dict[str, Any]] = {
+    "core_eiaaeo__generation_electric_sector": {
+        "description": (
+            "Projected generation capacity & total generation in the electric "
```

Comment:
It's not clear to me that generation and capacity *should* live as separate columns in the same table, especially since the "fuel type" subdimension is slightly different between the two data types - there are extra "sales to customers" and "generation for own use" values there which apply to the total generation but not the capacity.

I wonder if there should be "end-use/electricity-sector generation by fuel type" tables, and then also "end-use/electricity-sector generation *not by fuel type*" tables that include the "total across all fuel types" and "sales to customer/grid" and "generation for own use" columns.